libyul/Dialect.h

```diff
@@ -46,7 +46,7 @@ struct BuiltinFunction
 	std::vector<Type> returns;
 	SideEffects sideEffects;
 	ControlFlowSideEffects controlFlowSideEffects;
-	/// If true, this is the msize instruction.
+	/// If true, this is the msize instruction or might contain it.
```

Comment:
Why this change?

Reply:
because verbatim is not exactly the msize instruction, but it might contain it